### DISCUSSION TITLE
Gendex site

### DISCUSSION THREAD

**Discussion by ghost** [HuubMons/HuMo-genealogy#26] [Q&A] [answered]
Does this platform or application still exist? I can't find any info.

Do humogen users still use it?

**Comment by HuubMons** [comment, marked as answer]
Yes, a few users do use it. One of them is me :-)
The gendex script is connected to the (mainly dutch) website: https://www.stamboomzoeker.nl/

If any changes are done in the family tree in HuMo-genealogy, the results will be updated at the stamboomzoeker website.
